### PR TITLE
Feature/demo

### PR DESCRIPTION
**Pull request by bernardodesousa**

### CODE REVIEW

File: wiredanchor.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+.wiredanchor:hover,
+.wiredanchor:focus,
+.wiredanchor:visited,
+.wiredanchor:active {
+  text-decoration: none;
+  color: #ff6f6f;
+}
+
+.wiredanchor:hover::before{
```
Comment (felipebernardes) [inline wiredanchor.css]: add space before opening {

File: wiredanchor.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  opacity: 0;
+  transition: opacity 0.1s linear;
+  background-color: #ff6f6f;
+  width: 100%;
+  height: 82%;
+  position: absolute;
+  left: 0;
+  top: 0;
+}
+.wiredanchor:after {
```
Comment (felipebernardes) [inline wiredanchor.css]: add blank line between selectors

File: wiredanchor.css (hunk 1 above)
Comment (bernardodesousa) [reply wiredanchor.css]: Sure thing.

File: wiredanchor.css (hunk 2 above)
Comment (bernardodesousa) [reply wiredanchor.css]: Got it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Wiredanchor demo</title>
+    <link rel="stylesheet" href="wiredanchor.css">
+  </head>
+  <body>
+    <ul>
+      <li><h1><a class="wiredanchor default" href="#">Example 1</a></h1></li>
```
Comment (felipebernardes) [inline index.html]: remove unused default class

**Comment by felipebernardes** [comment]
GJ @bernardodesousa

**Pull request merged**